data-types.md

```diff
@@ -5,14 +5,14 @@ toc: false
 
 CockroachDB supports the following data types. Click a type for more details.
 
-Type | Description | Example
+Type | Description | Storage Size | Example
 -----|-------------|--------
-[`INT`](int.html) | A 64-bit signed integer. | `12345`
-[`DECIMAL`](decimal.html) | An exact, fixed-point number. | `DECIMAL '1.2345'`
-[`FLOAT`](float.html) | An inexact, floating-point number. | `1.2345`
-[`BOOL`](bool.html) | A Boolean value. | `true` 
-[`DATE`](date.html) | Year, month, day. | `DATE '2016-01-25'`
-[`TIMESTAMP`](timestamp.html) | A date and time pairing. | `TIMESTAMP '2016-01-25 10:10:10'`
-[`INTERVAL`](interval.html) | A span of time. | `INTERVAL '2h30m30s'`
-[`STRING`](string.html) | A string of characters. | `'a1b2c3'`
-[`BYTES`](bytes.html) | A string of binary characters. | `b'\141\061\142\062\143\063'`
\ No newline at end of file
+[`INT`](int.html) | 64-bit signed integer. | 8 bytes | `12345`
```

Comment:
Integers (and dates) are stored in a variable-length encoding, so they are 1-9 bytes.

I think it's misleading to give this level of detail (or at least to make it so prominent). There is significant per-row and per-field overhead, so you can't really use these values to estimate the total size of your database. And since we don't have multiple integer types, we don't need a table like this to explain the difference between `SMALLINT`, `INT`, and `BIGINT`